docs/tutorials/nodes-tutorial.ipynb

```diff
@@ -0,0 +1,376 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3dbc91a9-97a7-46e9-9c47-422a6bad5a94",
+   "metadata": {},
+   "source": [
+    "# Custom interactive interfaces\n",
+    "\n",
+    "This tutorial will demonstrate how to create custom interactive interfaces to visualize your data with `plopp`.\n",
+    "\n",
+    "## A set of connected nodes\n",
+    "\n",
+    "In `plopp`, think of each element in your interface as a set of interconnected nodes in a graph.\n",
+    "Each node can have parent nodes, children nodes, and also views attached to them (e.g. figures).\n",
+    "\n",
+    "At the most basic level, a graph will contain a node (white rectangle) that provides the input data,\n",
+    "and a view (grey ellipse) which will be figure to display the data visually.\n",
+    "\n",
+    "![graph](_static/node_graph.png)"
```

Comment:
`node_graph.png` does not show up in the docs. I get a warning from Sphinx:
```
/home/jl/Work/plopp/docs/tutorials/nodes-tutorial.ipynb:: WARNING: image file not readable: tutorials/_static/node_graph.png
```
The path here needs to be relative to the notebook.

Reply:
I re-organised the notebook so that I don't need the png anymore